app/domain/output_document.rb

```diff
@@ -21,7 +21,7 @@ def attendee_name
   end
 
   def appointment_date
-    appointment_summary.date_of_appointment.to_s(:long)
+    appointment_summary.date_of_appointment.strftime('%e %B %Y')
```

Comment:
I think its worth extracting this as a constant, and then adding a comment to its declaration that calls out why we're using this format, and links to the relevant bit of the GOV.UK service manual.